content/20.results.md

```diff
@@ -19,16 +19,31 @@ Because the concordance was high and the PubMed set was more complete, we used t
 
 ### Assessing Gender Diversity of Authors and Honorees
 
+Although Bioinformatics was established in 1998 and BMC Bioinformatics in 2000, the metadata for these journal papers before 2002 only have initials for first and/or middle author names.
+Therefore, without first and middle names, we do not have author gender predictions before this year.
+
+There is a slow increasing trend of the proportion of female authors over the years, arriving at just over 20% on average in 2019 (Fig. {@fig:gender_breakdown}A).
+Analyzing each journal individually, we observe very similar trend across the three selected journals, but the least increase in the estimated female proportion is found in PLOS Computational Biology (Supplementary Fig S1).
+ISCB fellows and keynote speakers appear to be more evenly split between men and women compared to the population of authors published in computational biology and bioinformatics journals (Fig. {@fig:gender_breakdown}B).
+The overall increasing female proportion trend seems to be present in each honor category, especially in the group of ISCB Fellows (Fig. {@fig:gender_breakdown}C).
+It is more difficult to make inference from the group of PSB speakers because the conference only has 1-2 keynotes each year (Fig. {@fig:gender_breakdown}C).
```

Comment:
```suggestion
PSB has only one or two keynotes per yet so it varies somewhat dramatically (Fig. {@fig:gender_breakdown}C), but the 2020 PSB keynotes were the only set of honorees across our analysis for which nearly all the probability was ascribed to female speakers.
There were many examples in which the probability of male speakers across the full set of honorees was above 95%.
```
I think the second part is right from the graph. Definitely 90%. Should check this to make sure it's accurate :)

Reply:
By examples, did you mean years?

A small typo on 'per year', but maybe two sentences would be clearer?

PSB has only one or two keynotes per year, so its estimated proportion of female speakers varies considerably (Fig. {@fig:gender_breakdown}C).
However, the 2020 PSB keynotes were the only set of honorees across our analysis for which nearly all the probability was ascribed to female speakers.

Reply:
I like that change! See what you think of this version (Edited the suggestion).

Reply:
I ended up taking out the comment about all male speakers too. It seems pretty apparent from the figure and maybe it's too heavy-handed to include it here.